src/lib.rs

```diff
@@ -63,6 +63,7 @@ use users::os::unix::UserExt;
 use openssh_keys::PublicKey;
 use std::fs::{File, self};
 use fs2::FileExt;
+use std::mem::drop;
```

Comment:
This should be part of the [prelude](https://doc.rust-lang.org/std/prelude/v1/index.html), so you shouldn't need this here. Any specific reason you added it?

Reply:
Nope, my memory just failed me :)